src/plugins/ipfs.ts

```diff
@@ -0,0 +1,33 @@
+/*eslint no-console: ["error", { allow: ["warn", "error"] }] */
```

Comment:
Why?

Reply:
There is a `console.error` in that file that prints an error on `fetch` fail

Reply:
ok, that works